ex_functions/csvParser.js

```diff
@@ -0,0 +1,48 @@
+const file=`Name;Street;City;Age
+Peter Pan;Am Hang 5;12345 Einsam;42
+Maria Schmitz;Kölner Straße 45;50123 Köln;43
+Paul Meier;Münchener Weg 1;87654 München;65`;
+
+function csvParser(file, separator, spaceBefore, spaceAfter){
+    if(typeof(file) != "string"){
+        throw Error("Insert file as string");
+    }
+
+    const rows = file.split(separator).map((line,index) => {
+        const columns = parseLine(line);
+        if(index == 0){
+            return columnsToHeader(columns, spaceBefore, spaceAfter);
+        }
+        return columnsToRow(columns, spaceBefore, spaceAfter);
+    });
+
+    return rows.join('<br>\r\n');
+}
+
+function columnsToHeader(columns, spaceBefore, spaceAfter){
```

Comment:
columnsToHeader fai 2 cousas `to header` e `to separator`, aínda que se podería interpretar que as 2 liñas son o `header`